.github/workflows/build.yml

```diff
@@ -0,0 +1,33 @@
+name: Build
+
+on:
+    workflow_dispatch:
+    push:
+        branches:
+            - main
+    pull_request:
+        branches:
+            - main
+
+jobs:
+    build:
+        name: Build
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
+
+            - name: Setup PNPM
+              uses: pnpm/action-setup@v2
+
+            - name: Setup node@20
+              uses: actions/setup-node@v3
+              with:
+                node-version: 20
+                cache: "pnpm"
+            
+            - name: Install dependencies
+              run: pnpm install
+
+            - name: Build
+              run: pnpm run build --build --verbose
```

Comment:
I'd prefer to move these flags to the `package.json` `build` script so we don't forget to keep them in sync.

Reply:
done